tutorials/auto_scheduler/tune_network_cuda.py

```diff
@@ -105,7 +105,6 @@ def get_network(name, batch_size, layout="NHWC", dtype="float32"):
         mod, params = relay.testing.squeezenet.get_workload(
             version="1.1",
             batch_size=batch_size,
-            layout=layout,
```

Comment:
why do we delete this?
We can add an assertion here if it does not support NHWC layout.

Reply:
Because the squeezenet get_network API doesn't have this argument.

Reply:
We can add `assert layout == "NCHW"` for this squeezenet